javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -39,6 +40,8 @@ function AppNavbar() {
           { (isMember || isAdmin)  &&
               <NavDropdown title="Search">
                   <NavDropdown.Item href="/member/messages/search">Message Search</NavDropdown.Item>
+                  <NavDropdown.Item href="/admin/students/team1">Manage Team1 Students</NavDropdown.Item>
+                  {/* <NavDropdown.Item href="/admin/students/team1">Manage Team1 Students</NavDropdown.Item> */}
```

Comment:
Why is this commented out code here?  Please remove.

Reply:
I believe this is fixed now!